src/GraphQL.Tests/Types/NonNullGraphTypeTests.cs

```diff
@@ -44,9 +44,12 @@ public void nonnullable_fields_without_values_do_complain()
                 expectedErrorCount: 3);
 
             var errors = result.Errors.ToArray();
-            errors[0].Message.ShouldBe("Cannot return null for non-null type. Field: a, Type: Int!.");
-            errors[1].Message.ShouldBe("Cannot return null for non-null type. Field: b, Type: Boolean!.");
-            errors[2].Message.ShouldBe("Cannot return null for non-null type. Field: c, Type: String!.");
+            errors[0].Message.ShouldBe("Error trying to resolve a.");
```

Comment:
Maybe it is worth to change base error message from `$"Error trying to resolve {node.Name}."` to `$"Error trying to resolve field '{node.Name}'."` in `SubscriptionExecutionStrategy` and `ExecutionStrategy` ?

Reply:
I agree.  It will probably change a million tests, so I'll put in a separate PR.

Reply:
OK